### DISCUSSION TITLE
Separation of data by Sample

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#971] [Questions / Documentation] [answered]
In my analysis of 2 datasets representing two age groups via Arch R I'm unable to write to file Peak-2-Gene links that are specific to each sample (groupBy="Sample")as opposed to groupBy="Clusters". 

Similarly, I was wondering how i could obtain separate plots Identification of Positive Transcription Factors (chapter 15.4 in full manual).

I started of 
seGroupMotif <- getGroupSE(ArchRProj = proj5, useMatrix = "MotifMatrix", groupBy = "Sample")
However, proceeding from here I do not know what to change to get two plots: 1 for each sample.
Can you assist with the script needed to achieve this?

Thanks!

**Comment by rcorces** [comment, marked as answer]
Why not create two different project subsets using `subsetArchRProject()`?

**Comment by sylestiel** [comment]
Hmm interesting. 
So follow the following script:
idxSample <- BiocGenerics::which(projHeme1$Sample %in% "scATAC_BMMC_R1")
cellsSample <- projHeme1$cellNames[idxSample]
projHeme1[cellsSample, ]

Would it not be the same as doing the analysis for each one separately?
How does normalization work in that case and also when the two are processed together?

**Reply by rcorces** [reply]
we recommend using the `subsetArchRProject()` function whenever subsetting a project because this is more stable.

> How does normalization work in that case and also when the two are processed together?

I'm not sure what you mean by normalization.

**Comment by sylestiel** [comment]
How is **subsetArchRProject()** different from 
**idxSample <- BiocGenerics::which(projHeme1$Sample %in% "scATAC_BMMC_R1")
cellsSample <- projHeme1$cellNames[idxSample]
projHeme1[cellsSample, ]**

Also would I need to repeat LSI_ATAC, LSI_RNA again or is it okay to proceed with the original LSI ?


Also how is this approach of combining different conditions in one analysis similar to merging or integration of data in Signac?

Thanks!

**Reply by rcorces** [reply]
Its preferable if you reply within the same conversation rather than adding a new comment.

`subsetArchRProject()` ensures that the data in your Arrow files matches exactly the expected data in your ArchRProject.
subsetting using [] does not alter the Arrow files at all, just the ArchRProject.
See the code if you need further clarification

> Also would I need to repeat LSI_ATAC, LSI_RNA again or is it okay to proceed with the original LSI ?

This depends on your application. you dont *need* to.

> Also how is this approach of combining different conditions in one analysis similar to merging or integration of data in Signac?

I dont know how to answer that question.

**Comment by sylestiel** [comment]
Thank you!
  
   On August 13, 2021 at 6:13 PM Ryan Corces ***@***.***> wrote: 
    
   
    
   Its preferable if you reply within the same conversation rather than adding a new comment.
   subsetArchRProject() ensures that the data in your Arrow files matches exactly the expected data in your ArchRProject. subsetting using [] does not alter the Arrow files at all, just the ArchRProject. See the code if you need further clarification
   
    Also would I need to repeat LSI_ATAC, LSI_RNA again or is it okay to proceed with the original LSI ?
   
   This depends on your application. you dont need to.
   
    Also how is this approach of combining different conditions in one analysis similar to merging or integration of data in Signac?
   
   I dont know how to answer that question.
